intents.json

```diff
@@ -211,9 +214,9 @@
                 "What's the forecast for the week"
             ],
             "responses": [
-                "Sure! What city would you like the weather forecast for?",
-                "I'd be happy to check the weather. Could you please tell me the city?",
-                "Great! To provide the weather forecast, I just need to know the city name. What city are you interested in?"
+                "The current temperature in {city} is {temperature} degrees Celsius. And, currently the weather is {description}.",
```

Comment:
Change the temperature responses to previous

```json
"responses": [
                "Sure! What city would you like the weather forecast for?",
                "I'd be happy to check the weather. Could you please tell me the city?",
                "Great! To provide the weather forecast, I just need to know the city name. What city are you interested in?"
]
```

Reply:
changed temperature response to previous